rainbow-me/rainbow
# Cleanup unused Savings components and related logic

What to test:
* Confirm showcase sheet works as expected
* Wallet screen should also work as expected (can test around with wallets that have NFTs vs positions vs assets vs empty, etc)
* Send sheet should still have a separate area for cTokens
